src/BenchmarksDriver/Program.cs

```diff
@@ -87,6 +87,9 @@ public static int Main(string[] args)
                 "Source dependency. Format is 'repo@branchOrCommit'. " +
                 "Repo can be a full URL, or a short name under https://github.com/aspnet.",
                 CommandOptionType.MultipleValue);
+            var threadCountOption = app.Option("-t|--threadCount",
+                "Kestrel ThreadCount.",
+                CommandOptionType.SingleValue);
```

Comment:
Bit confusing. Suggest we rename `--threads` to `--clientThreads` and make this one `--serverThreads` or perhaps `-c|--clientthreads` and `-s|--serverThreads`. I'd also rather we use `-t` with `--duration` but we can separate that.